src/stores/features.js

```diff
@@ -36,17 +36,20 @@ export const useFeaturesStore = defineStore('features', () => {
 
   const collection = computed(() => {
```

Comment:
```suggestion
  /**
   * The intent of this computed collection is to *always* provide a CPF value to features.
   * 1. Because they are explitly set from a controlled-list via the UI (the `is_selectable` crop attribute)
   * 2. Because they are translated from a PAC code (the `TYPE` feature property)
   * 3. Or because they are set from an external system (the crop may or may not be selectable via the UI)
   */
  const collection = computed(() => {
```